### DISCUSSION TITLE
3d Printed Support

### DISCUSSION THREAD

**Discussion by rrlevy** [tesla-android/issue-tracker#84] [Ideas]
I'm working on designing a 3d-printed support for the 2 Raspiberry Pi, Modem and Carlinkit.

This is the current state:

![Tesla Android Support v15](https://user-images.githubusercontent.com/14061585/188368847-3a6bde5f-8534-45e1-946a-4702a9127730.png)

I will share 3d files if it works

**Comment by rrlevy** [comment]
Here is the final version for the two-board stack:

![E24303B2-569C-40C2-B7D8-A52F0A4C8A09](https://user-images.githubusercontent.com/14061585/189762501-0dc805df-dc1c-4e6f-8878-23521a6ca274.jpeg)

![150844E2-4DC8-4EF7-8DB0-6E5620CA9EF3](https://user-images.githubusercontent.com/14061585/189762526-f5732a53-722b-47ea-bd21-a91ddb2a201e.jpeg)

![1706DC53-2456-4829-8848-5D3AE950CA14](https://user-images.githubusercontent.com/14061585/189762539-f663c123-43bb-4575-bd6c-1fec1023870e.jpeg)

![4D85D5E5-F7F3-43DF-9DC1-840B6140CF33](https://user-images.githubusercontent.com/14061585/189762600-45100116-fe48-40c1-85d2-6d9e148697ea.jpeg)

I will add it to Thingiverse and post the link here

**Reply by rrlevy** [reply]
Here is the link for download:

https://www.thingiverse.com/thing:5510443

**Reply by martindell** [reply]
ace job, thank you!
